include/swift/AST/DiagnosticsSema.def

```diff
@@ -2089,7 +2089,7 @@ ERROR(property_behavior_protocol_no_initStorage,none,
       (Type, Type))
 ERROR(property_behavior_unknown_requirement,none,
       "property behavior protocol %0 has non-behavior requirement %1",
-      (Identifier, Identifier))
+      (Identifier, DeclBaseName))
```

Comment:
OK, I'll change it to `DeclName` then. Just thought it would be best to keep the hidden parts up-to-date as well.